src/components/core/base/GeoValidatorHOC.tsx

```diff
@@ -80,6 +83,25 @@ export function geoValidatorHOC<T>(InnerComponent: React.ComponentClass<T>): Rea
 
             return <ErrorComponent code={errorState} {...errorProps} />;
         }
+
+        private isSameConfig(config: IGeoConfig, nextConfig: IGeoConfig): boolean {
+            const colorMapping = (config.colorMapping || []).map(
+                (currentColor: IColorMapping) => currentColor.color,
```

Comment:
type for predicate :)
`(currentColor: IColorMapping): IColorItem => currentColor.color,`

Reply:
updated